challenges/objects-arrays.js

```diff
@@ -43,10 +62,15 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
+
 const universities = [];
-console.log(universities)
+
+
+for(let i= 0; graduates.length > i; i++) {
+  universities.push(graduates[i].university);
+}
+console.log(universities);
```

Comment:
Don't forget you need to sort the universities alphabetically! `console.log(universities.sort())`